### PR TITLE
Ensure keys are present else return 500

### PR DESCRIPTION
**Pull request by admiral96**
Mingle Card Ref: #4152
    https://engagespark.mingle.thoughtworks.com/projects/es/cards/4152

### CODE REVIEW

**Comment by dashmug** [comment]
You should not be checking for API keys in the views. 

Your application should check for the presence of those API keys on startup. If the API keys are not available, it should be a fatal error and your application SHOULD refuse to run.

File: app/geocode_api_parser.py [hunk 1]
```diff
@@ -32,3 +32,6 @@ def get_place_info(self):
         self.formatted_address = geocode_info['results'][0]['formatted_address']
         self.place_type = geocode_info['results'][0]['address_components'][0]['types'][0]
         return {'place_type': self.place_type, 'formatted_address': self.formatted_address}
+
+    def check_api_is_available(self):
```
Comment (dashmug) [inline app/geocode_api_parser.py]: You are supposed to be checking for the presence of the _API keys_ not availability of the _API_.

**Comment by mknecht** [comment]
@dashmug true re not checking in the view. but since there are _other webhooks_ in the app, too, it should continue running to serve those, log a horrible warning and return a 500 on this particular webhook that needs the keys.

**Comment by admiral96** [comment]
@mknecht Do I still have something to do in this PR?

**Pull request merged**